OpenRA.Mods.Common/Traits/Buildings/Building.cs

```diff
@@ -154,7 +154,11 @@ public WVec CenterOffset(World w)
 		public Actor FindBaseProvider(World world, Player p, CPos topLeft)
 		{
 			var center = world.Map.CenterOfCell(topLeft) + CenterOffset(world);
-			var allyBuildEnabled = world.WorldActor.Trait<MapBuildRadius>().AllyBuildRadiusEnabled;
+			var mapBuildRadius = world.WorldActor.Trait<MapBuildRadius>();
+			var allyBuildEnabled = mapBuildRadius.AllyBuildRadiusEnabled;
+
+            if (!mapBuildRadius.BuildRadiusEnabled)
```

Comment:
These lines are now indented using spaces.  Please replace them with tabs.

https://github.com/OpenRA/OpenRA/wiki/Coding-Standard includes a guide on how to configure Visual Studio to use tabs.

Reply:
but I use linux